pkg/secrethub/service.go

```diff
@@ -115,3 +118,52 @@ func (s serviceService) List(path string) ([]*api.Service, error) {
 	repoServiceService := newRepoServiceService(s.client)
 	return repoServiceService.List(path)
 }
+
+// Iterator returns an iterator that lists all service accounts in a given repository.
```

Comment:
Another case where the comment is still there from a copy-paste. Could you check all iterators to see if there are more cases?